python/tk_desktop/desktop_window.py

```diff
@@ -135,7 +133,9 @@ def __init__(self, parent=None):
         self.user_menu.addAction(self.ui.actionRefresh_Projects)
         about_action = self.user_menu.addAction("About...")
         self.user_menu.addSeparator()
-        self.user_menu.addAction(self.ui.actionSign_Out)
+        # Only show sign out if you are logged as a human user.
+        if sg_auth.is_session_user(sgtk.get_current_user()):
```

Comment:
Is this possible? Could you be logged in as something else?
I also thought we decided to try to just have a user object and not distinguish between a script user and a human user outside of the security API?